04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Jdaviz.ipynb

```diff
@@ -9,8 +9,8 @@
    "source": [
```

Comment:
<p>Starting at DESI-DR1, the <code>desiname</code> field is included in SPARCL as an AUX field for SPARCL. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>OK, I should have time to update that.</p><p> </p>